felles/src/main/kotlin/no/nav/familie/kontrakter/felles/simulering/SimulertPostering.kt

```diff
@@ -5,6 +5,7 @@ import java.time.LocalDate
 
 data class SimulertPostering(
     val fagOmrådeKode: FagOmrådeKode,
+    val erFeilkonto: Boolean?,
```

Comment:
Kan du skrive et liten kommentar for å beskrive hvorfor dette feltet blir introdusert? Siden det kommer til å påvirke alle familie ytelser, er det beste å ha ett kommentar på den